dev-docs/batch-operation.md

```diff
@@ -33,3 +38,26 @@ Watch the behavior under maximum load.  Normally the allocated cores increases f
 You can inspect the `internal-gateway` logs to determine if the request rate is  maximized.  When the maximum request rate is exceeded, `internal-gateway` nginx returns 503 and logs a message.
 
 To determine if the cluster size is at the maximum, check the CPU and `internal-gateway` request rate when the cluster is not growing, but just replacing preempted nodes.  The CPU should not be pegged, and `internal-gateway` should reject requests at most in transient bursts.  In general, the load will be much lower at equilibrium because filling an empty node requires many operations.
+
+## Quotas
+
+When using Local SSDs on preemptible machines there are only two quotas that matter: "Preemptible
+Local SSD (GB)" (`PREEMPTIBLE_LOCAL_SSD_GB`) and "Preemptible CPUs" (`PREEMPTIBLE_CPUS`). The former
+is measured in GB, so you'll need 375 GB of quota for every machine with a Local SSD. The latter is
+measured in cores. For example, if you are using a mix of n1 and n2 machines with 8 cores and 1
+Local SSD, a 5000 machine (40,000 core) cluster will need:
+
+- 1,875,000 GB of Preemptible Local SSD quota, and
+
+- 40,000 cores of Preemptible CPUs quota.
+
+In practice, we use Local SSD quota much faster than CPU quota. Google will freely gives us a 5,000
+core quota in any given zone. We've also received quotas as high as 300,000 cores. Google is
+hesitant to grant a quota of more than 400,000 GB in a zone. The largest Preemptible Local SSD quota
+we have been granted in one zone is 640,000 GB.
+
+We recommend requesting double your quota when you're using 80-90% of the current quota. Repeating
+this process will generally allow you to quickly scale the cluster.
```

Comment:
Could you leave one more sentence specifying that requesting a quota just entails `Edit`ing the quota in the google console?